tutorials/getting_started/rules_advanced.md

```diff
@@ -208,6 +338,30 @@ var endTime = time.ZonedDateTime.now().withHour(23).withMinute(0).withSecond(0).
 now.isAfter(sunset) && now.isBefore(endTime)
 ```
 
+:::
+
+::: tab JRuby
+
+```ruby
+Time.now.between? Sunset.state.to_s.."23:00"
+
+# Alternative code:
+# ZonedDateTime.now.between? Sunset.state.to_s.."23:00"
+# Time.now.between? Sunset.state, Time.parse("23:00") # Two arguments of date/time objects
+# Time.now >= Sunset.state && Time.now <= Time.parse("23:00")
+# etc.
+```
+
+[#between?](https://openhab.github.io/openhab-jruby/main/OpenHAB/CoreExt/Between.html#between%3F-instance_method) helper method is available on all Ruby and Java date, time, and Duration objects.
+It takes a Ruby [Range](https://docs.ruby-lang.org/en/master/Range.html) as its argument, or two Date/Time objects to perform an inclusive range comparison.
+
+Note that in JRuby, Date/Time and also DateTimeType objects (in the above example, the item's state) can all work interchangeably without any explicit conversions.
```

Comment:
```suggestion
Note that in JRuby, Ruby Time, Java ZonedDateTime and Instant, openHAB DateTimeType type (in the above example, the item's state), and several other time related types can all work interchangeably without any explicit conversions.
```

Note that I explicitly left out Ruby's Date type - while it exists and is compatible, its use is highly discouraged, and so shouldn't be mentioned in such a tutorial.